strictdoc/export/html/form_objects/document_grammar_form_object.py

```diff
@@ -136,11 +142,15 @@ def create_from_request(
         document_grammar_fields = request_form_dict["document_grammar_field"]
         for field_mid, field_dict in document_grammar_fields.items():
             field_name = field_dict["field_name"]
+            field_human_title = field_dict.get("field_human_title")
+            if field_human_title is not None and not field_human_title.strip():
```

Comment:
Here, let's not do several things on one line. I would prefer it this way:

```py
if field_human_title is not None:
    field_human_title = field_human_title.strip()
    if len(field_human_title) == 0:
        field_human_title = None
```

Note that the `.strip` is important for both empty and non-empty string cases, and my example ensures that.